### PR TITLE
add eBPF_proc_image

### PR DESCRIPTION
**Pull request by zhangzihengya**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: eBPF_Supermarket/eBPF_proc_image/test/test_A.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+#include <unistd.h>
```
Comment (helight) [inline eBPF_Supermarket/eBPF_proc_image/test/test_A.c]: 这个测试文件好像没有使用，可以把这个测试文件用起来，让测试更加完善

Comment (helight) [reply eBPF_Supermarket/eBPF_proc_image/test/test_A.c]: 另外最好给这个 c 文件也增加 makefile

Comment (zhangzihengya) [reply eBPF_Supermarket/eBPF_proc_image/test/test_A.c]: 师兄，这个就是一个简单的c测试程序，用于验证我的eBPF程序的正确性，需要两个终端以及我的手动输入才能实现操作，在github上实现不了这样的测试，我已经在readme上把这个测试进行了描述

Comment (chenamy2017) [reply eBPF_Supermarket/eBPF_proc_image/test/test_A.c]: 每一个测试程序的文件名称可以具体点。

Comment (zhangzihengya) [reply eBPF_Supermarket/eBPF_proc_image/test/test_A.c]: > 每一个测试程序的文件名称可以具体点。

收到，已改

**Review by helight — APPROVED** [review]
LGTM

**Pull request merged**